src/Stratis.Bitcoin.Networks/Stratis.Bitcoin.Networks.csproj

```diff
@@ -20,6 +20,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\NBitcoin\NBitcoin.csproj" />
+    <ProjectReference Include="..\Stratis.Bitcoin.Features.Consensus\Stratis.Bitcoin.Features.Consensus.csproj" />
```

Comment:
Ah why do we need to reference this? I think that this assembly should only reference NBitcoin?

Reply:
No this is ok to reference lower projects from here especially that now rules go in the network (most of the rules actually currently exist in network)